docs/docsite/rst/user_guide/playbooks_loops.rst

```diff
@@ -1,21 +1,40 @@
 .. _playbooks_loops:
 
+*****
 Loops
-=====
+*****
 
-Often you'll want to do many things in one task, such as create a lot of users, install a lot of packages, or
-repeat a polling step until a certain result is reached.
+Sometimes you want to repeat a task multiple times. In computer programming, this is called a loop. Common Ansible loops include installing several packages with the :ref:`yum module <yum_module>`, creating multiple users with the :ref:`user module <user_module>`, and
+repeating a polling step until a certain result is reached. Ansible offers two keywords for creating loops: ``loop`` and ``with_<lookup>``.
 
-This chapter is all about how to use loops in playbooks.
+.. note::
+   * We added ``loop`` in Ansible 2.5, but it is not yet a full replacement for ``with_<lookup>``.
+   * We have not deprecated the use of ``with_<lookup>``.
+   * We are still discussing what UX changes can be made to enable ``loop`` to replace ``with_<lookup>`` in the future.
+   * If we eventually deprecate the ``with_`` syntax, the deprecation cycle will be longer than usual.
 
-.. contents:: Topics
+.. contents::
+   :local:
+
+Comparing ``loop`` and ``with_*``
+=================================
+
+* The ``with_<lookup>`` keywords rely on :ref:`lookup_plugins` - even  ``items`` is a lookup.
+* The ``loop`` keyword is equivalent to ``with_list``, and is the best choice for simple loops.
+* The ``loop`` keyword will not accept a string as input, see :ref:`query_vs_lookup`.
+* Generally speaking, any use of ``with_*`` covered in :ref:`migrating_to_loop` can be updated to use ``loop``.
+* Be careful when changing ``with_items`` to ``loop``, as ``with_items`` performed implicit flattening. You may need to use ``flatten`` with ``loop`` to match the exact outcome.
+* Any ``with_*`` statement that requires using ``lookup`` within a loop should not be converted to use the ``loop`` keyword.
```

Comment:
this last one could be rephrased  to ' if adding a single lookup to loop, you with_<lookup> will give you a cleaner syntax, once you have multiple lookups, you probably are better off with loop

Reply:
Can you show an example? If using `lookup` once within an iterative task means you should use `with_*` instead of `loop`, I would expect more uses of `lookup` would make things worse with `loop` . . . but it's easier to think about in the context of a concrete example.

Reply:
@bcoca @sivel can you help out with examples of `with_*` constructions using `lookup` that should / should not be changed to `loop`? Or more clarity/discussion around this single lookup / multiple lookups question?

Reply:
How about - Any `with_*` statement that requires using  more than one `lookup` within a loop should  be converted to use the `loop`keyword.   You don't need to change a `with_*` with a single lookup or any use of  `with_list`.

(the with_list part might be digging to far into the details for a bullet list item?)